src/operation.js

```diff
@@ -438,31 +430,33 @@ export class Operation {
         }
 
         let attrs = operation.body().value();
+        let n, price;
         switch (operation.body().switch().name) {
             case "createAccount":
                 result.type = "createAccount";
                 result.destination = accountIdtoAddress(attrs.destination());
-                result.startingBalance = attrs.startingBalance().toString();
+                result.startingBalance = new BigNumber(attrs.startingBalance()).div(ONE).toString();
```

Comment:
Whole 'nother round of copy-pasta going on here.   Extract to a function called `_fromStellarAmount`.